service/initialise.go

```diff
@@ -49,6 +50,8 @@ func (e *ExternalServiceList) GetRequestMiddleware() RequestMiddleware {
 // DoGetHTTPServer creates an HTTP Server with the provided bind address and router
 func (e *Init) DoGetHTTPServer(bindAddr string, router http.Handler) HTTPServer {
 	s := dphttp.NewServer(bindAddr, router)
+	s.Server.ReadTimeout = 15 * time.Second
+	s.Server.WriteTimeout = 30 * time.Second
 	s.HandleOSSignals = false
```

Comment:
how sure are we on these? Would we want these to be config vars?

Reply:
We've run JMeter tests with these value as their default (read 5s, write 10s) and the error found on the logs from performance testing is appearing locally. With this increase, the error is no longer present locally. Just going to lower them again and find the point where the errors start to persist.

Reply:
seems like if we're not sure, then altering via config seems a sensible approach.

Reply:
Have added these values to the config